Dockerfile

```diff
@@ -1,12 +1,20 @@
-FROM alpine:3.4
+FROM golang:1.9 as builder
 MAINTAINER Travis CI GmbH <support+travis-worker-docker-image@travis-ci.org>
 
-ADD build/linux/amd64/travis-worker /usr/local/bin/travis-worker
-ADD .docker-entrypoint.sh /docker-entrypoint.sh
+RUN go get -u github.com/FiloSottile/gvt
 
-RUN apk add --no-cache ca-certificates curl bash
+COPY . /go/src/github.com/travis-ci/worker
+WORKDIR /go/src/github.com/travis-ci/worker
+RUN make deps
+RUN make build
```

Comment:
I'm pretty sure the binary generated here will not work inside an alpine container as-is, which is why the separate `make crossbuild` target was previously used.  I think you should be able to get the same end result by adding a directive above of `ENV CGO_ENABLED 0`.

Reply:
Thanks! Added.